### ISSUE TITLE
Shift overrides numlock

### ISSUE THREAD

**Issue by Talicrons** [ppy/osu-stable-issues#589]
When shift is pressed it overrides numlock and turns it off. So pressing shift and numpad1 results in End being pressed. This becomes a problem when I play Catch and I have my movement keys bound to num1 and num3. Accidentally hitting shift during gameplay would result in the catcher 'locking up' to one side. Besides the obvious solution of just don't hit shift, it would be nice if I could bound the dash key to shift as well.

A workaround to this is to play with numlock off and bound the keys to end and pagedown. But that involves toggling numlock every time I open and close osu, which is becoming quite annoying. 

I know this is inherent behavior in Windows, but it would be nice if the game could use keyboard scan codes and bypass this behavior.

**Comment by peppy** [comment]
Have you tested this in lazer? Does it behave as you'd hope? I doubt we will fix this in stable, but it may be something we can work on for the future.

**Comment by Talicrons** [comment]
Yes this is still an issue in lazer. It's not a big deal, I just wanted to make the issue known as I had spent a long time trying to figure out why the catcher locks up sometimes.
